pyvista/plotting/renderer.py

```diff
@@ -94,8 +94,8 @@ def scale_point(camera, point, invert=False):
     camera : Camera
         The camera who's matrix to use.
 
-    point : tuple(float)
-        Length 3 tuple of the point coordinates.
+    point : sequence[float, float, float]
```

Comment:
Bunch of these all across this module as well...
```suggestion
    point : sequence[float]
```